components/SearchBar.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * A React component that is renders the search bar
+ * @returns SearchBar
+ */
+import React, { useEffect, useState } from 'react';
+import {
+  TextField,
+  Typography,
+  ListItemText,
+  Avatar,
+  Grid,
+  Autocomplete,
+} from '@mui/material';
+import PropTypes from 'prop-types';
+
+const PLACEHOLDER_IMAGE =
+  'https://cdn4.buysellads.net/uu/1/127419/1670531697-AdobeTeams.jpg';
+const ITEMS_ON_SEARCH_LIST = 5;
+
+const SearchResultItem = ({ item }) => (
+  <Grid
+    container
+    spacing={2}
+    alignItems='center'
+    justifyContent='left'
+    paddingLeft={2}
+  >
+    <Grid item>
+      <Avatar src={item.imgpic} alt={item.name} />
+    </Grid>
+    <Grid item xs={6}>
+      <ListItemText primary={item.name} secondary={item.type} />
+    </Grid>
+    <Grid item xs={3}>
+      <ListItemText
+        primary={item.total_quantityopen}
+        secondary={item.total_quantityunopened}
```

Comment:
should we add some icons? so that its clearer what the number means

code does seem a little ugly tho (couldn't find any suitable icons in MUI)

<img width="695" alt="Screenshot 2023-06-16 at 12 34 58 AM" src="https://github.com/sjb-deck/deck/assets/55698556/b00406bb-ed36-4033-8ebd-981992e2da2f">


```suggestion
       primary={item.total_quantityopen + item.total_quantityunopened}
        secondary={
          <span>
            <svg
              xmlns='http://www.w3.org/2000/svg'
              width='16'
              height='16'
              fill='currentColor'
              className='bi bi-dropbox'
              viewBox='0 0 16 16'
            >
              <path d='M8.01 4.555 4.005 7.11 8.01 9.665 4.005 12.22 0 9.651l4.005-2.555L0 4.555 4.005 2 8.01 4.555Zm-4.026 8.487 4.006-2.555 4.005 2.555-4.005 2.555-4.006-2.555Zm4.026-3.39 4.005-2.556L8.01 4.555 11.995 2 16 4.555 11.995 7.11 16 9.665l-4.005 2.555L8.01 9.651Z' />
            </svg>
            {': ' + item.total_quantityopen}
            <br />
            <svg
              xmlns='http://www.w3.org/2000/svg'
              width='16'
              height='16'
              fill='currentColor'
              className='bi bi-box-seam-fill'
              viewBox='0 0 16 16'
            >
              <path
                fillRule='evenodd'
                d='M15.528 2.973a.75.75 0 0 1 .472.696v8.662a.75.75 0 0 1-.472.696l-7.25 2.9a.75.75 0 0 1-.557 0l-7.25-2.9A.75.75 0 0 1 0 12.331V3.669a.75.75 0 0 1 .471-.696L7.443.184l.01-.003.268-.108a.75.75 0 0 1 .558 0l.269.108.01.003 6.97 2.789ZM10.404 2 4.25 4.461 1.846 3.5 1 3.839v.4l6.5 2.6v7.922l.5.2.5-.2V6.84l6.5-2.6v-.4l-.846-.339L8 5.961 5.596 5l6.154-2.461L10.404 2Z'
              />
            </svg>
            {': ' + item.total_quantityunopened}
          </span>
        }
```